package.json

```diff
@@ -1,7 +1,18 @@
 {
   "name": "webext-firstpartyisolation",
   "version": "1.3.1",
+  "private": true,
```

Comment:
Why is this necessary? :-) First time I see this

Reply:
Since the package isn't published on npm, this flag prevents accidental publishing.